core/sail/fts/lucene-api/src/main/java/org/eclipse/rdf4j/sail/lucene/LuceneSail.java

```diff
@@ -419,6 +459,40 @@ public void setIncompleteQueryFails(boolean incompleteQueryFails) {
 	}
 
 	/**
+	 * See EVALUATION_MODE_KEY parameter.
+	 */
+	public String getEvaluationMode() {
```

Comment:
For clarity I think it might be good to create an `Enum` for the various Evaluation modes.